zerver/lib/migrate.py

```diff
@@ -1,8 +1,11 @@
 from __future__ import print_function
+from typing import *
+from django.db.models.query import QuerySet
 import re
 import time
 
 def timed_ddl(db, stmt):
+    # type: (Any, str) -> None # TODO: type of db?
```

Comment:
This file is unused because it's from Django south migrations (which ceased existing in Django 1.7 when migrations became a built-in feature of Django).  So the type of db was a `south.db.db` object.  I think it will be possible and valuable to eventually translate this file to work with the new Django 1.7+ built-in migrations stuff, and the interface is likely similar, so I'm OK with not deleting the file.  The upshot is it's probably fine to leave the type as `Any` here.